style.css

```diff
@@ -1 +1,347 @@
-/*  Add your styling here */
+:root {
+    --bg: #f3f0ef;
+    --text: #4c4946;
+    --yellow: #f7e259;
+    --yellowLight: #fff9de;
+    --ff-body: 'Open Sans', sans-serif;
+    --ff-secondary: 'Gwendolyn', cursive;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+}
+
+html, body {
+    min-height: 100vh;
+}
+
+html {
+    font-size: 16px;
+}
+
+body {
+    font-family: var(--ff-body);
+    -webkit-font-smoothing: antialiased;
+    font-weight: 300;
+    color: black;
+    background: var(--bg);
+    line-height: 1.4;
+}
+
+h1, h2, h3, h4, h5 {
+    font-weight: 500;
+    margin: 0;
+}
+
+/** General  */
+a {
+    display: inline-block;
+    color: inherit;
+}
+
+nav a {
+    text-decoration: none;
+}
+
+ul {
+    list-style-type: none;
+    padding: 0;
+}
+img {
+    display: inline-block;
+    max-width: 100%;
+    height: auto;
+}
+
+p {
+    font-size: 1.07rem;
+    line-height: 1.8;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.section_title {
+    font-size: 3rem;
+    margin-bottom: 1rem;
+    font-weight: 500;
+}
+
+.section_subtitle {
+    font-size: 2rem;
+    margin-bottom: 1rem;
+    font-weight: 500;
+}
+
+.wrapper {
+    display: grid;
+    min-height: 100vh;
+    grid-template-areas:
+        "header"
+        "welcome"
+        "about"
+        "gallery"
+        "footer";
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(5, auto);
+}
+
+section {
+    padding-top: 10px;
+    padding-bottom: 10px;
+}
+
+/*    HEADER    */
+.header {
+    grid-area: header;
+    padding: 20px;
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.content{
+    padding: 20px;
+}
+
+.main {
+    grid-area: main;
+    flex-grow: 1;
+}
+
+.menu_icon{
+    display: flex;
+    flex-direction: column;
+    row-gap: 5px;
+    align-items: center;
+    justify-content: center;
+    z-index: 101;
+    position: relative;
+}
+
+.menu_icon:hover{
```

Comment:
Because you're controlling the cursor, you can simplify this to remove the hover state. It won't apply in any other state anyway

```suggestion
.menu_icon {
```